pkg/poc/frontend.go

```diff
@@ -51,15 +56,34 @@ func (f *frontend) Run(ctx context.Context) error {
 	return err
 }
 
-func (f *frontend) getRouter() chi.Router {
+func (f *frontend) getRouter(ctx context.Context) chi.Router {
 	r := chi.NewRouter()
 	r.Get("/", func(w http.ResponseWriter, r *http.Request) {
 		f.logger.Infof("Received request: %s", time.Now().String())
-		w.Write([]byte("****** ARO-RP on AKS PoC ******"))
+		// TODO(jonachang): remove this when go production.
+		if f.enableMISE == true {
+			miseToken := extractMISEToken(r.Header)
+			miseError := authenticateWithMISE(ctx, miseToken)
+			if miseError != nil {
+				message := fmt.Sprintf("MISE error: %s", miseError)
+				f.logger.Info(message)
```

Comment:
To  make this simpler, you can just do `f.logger.Infof("MISE error: %s, miseError)`